### ISSUE TITLE
enhancement: remove flask-apispec

### ISSUE THREAD

**Issue by olevski** [SwissDataScienceCenter/renku-notebooks#631]
This was already done for renku-python. Follow the same approach.

https://github.com/SwissDataScienceCenter/renku-python/pull/2040
https://github.com/SwissDataScienceCenter/renku/pull/2091

depends on #622

**Comment by olevski** [comment]
This has been done already with https://github.com/SwissDataScienceCenter/renku-notebooks/pull/894
